src/options/options_handler.cpp

```diff
@@ -277,22 +277,23 @@ void OptionsHandler::setStats(const std::string& option, bool value)
 #endif /* CVC5_STATISTICS_ON */
   if (value)
   {
-    if (option == options::statisticsAll.getName())
+    if (option == std::string("--") + options::statisticsAll.getName())
```

Comment:
Nit: this is a bit ugly, can we do option.substr(2) on the left hand side instead? And moreover store this a temp variable above?  I'm actually surprised this method isn't stripping this earlier.

Reply:
I'd rather not do the `substr(2)`. It is not more efficient (both variants create a temporary), but it may do weird stuff if `option` for some reason does not start with `--`. I felt this version is clearer and more robust.

As for the temp variable: what exactly do you want to put in a temp variable?

Reply:
I'm saying that this method should ideally be refactored later so that option contains `"stats"` only.
The temporary would ensure this is stored at the beginning of this method (or at the beginning of the block where it is used).

Reply:
Ah, I see.